### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by iamiori** [1Panel-dev/1Panel#6903] [Q&A]
### 1Panel 版本

v1.10.20-lts

### 问题描述

主机》终端       无法使用了。更新后。
提示  
disconnected
The connection has been disconnected.

Linux 服务器运维管理面板
社区版  v1.10.20-lts

### 重现步骤

什么机子都连不上

**Comment by wanghe-fit2cloud** [comment]
参考：
https://github.com/1Panel-dev/1Panel/issues/3852
https://github.com/1Panel-dev/1Panel/issues/5703
https://github.com/1Panel-dev/1Panel/issues/4641
